NEWS

```diff
@@ -11,6 +11,211 @@
+ * Detected device inter-node capability with a new attribute flag
+ * Enabled using CUDA staging buffers for pipeline protocols by default
+ * Added endpoint reconfiguration support for non-reused p2p scenarios
+ * Enabled non-cacheable memory domains, activated for GDR
+ * Improved message upon wireup failure
+ * Added new protocol infrastructure to store aggregated factors for all ranges
+ * Added user_data parameter to ucp_ep_query
+ * Added support for host memory pipeline through CUDA buffers for rendezvous protocol
+ * Added global VA infrastructure and memory region in absence of error handling
+ * Improved performance nodes naming to be more informative
+ * Enforced always running on the same thread in single thread mode
+ * Used registration cache overhead from environment variable
+ * Improved protocol selection by aggregating multiple performance segments
+ * Added UCP_MEM_MAP_BLOCK API flag to enforce blocking memory mapping
+ * Allowed up-to 64 lanes for systems with many transports or devices
+ * Added rendezvous performance variants
+ * Added usage tracker to worker
+ * Added support for different variants of rendezvous protocol
+ * Multiple protocol selection algorithm improvements
+#### UCT
+ * Improved API documentation for memory domain flags
+#### RDMA CORE (IB, ROCE, etc.)
+ * Enabled support for a broader range or DC initiator
+ * Added DC dcs_hybrid policy
+ * Reduced MLX5/DV stack size consumption
+ * Improved MLX5 log level when allocations are unsupported, for both WC and NC
+ * Added ODP support for verbs and mlx5dv
+ * Added environment variable to enable setting DC initiator capacity
+ * Enabled DDP support on mlx5dv
+ * Added support of cuda-managed memory on IB when ODP is available
+ * Enabled use of implicit ODP with relaxed ordering
+ * Improved GPUDirect detection support on IB
+ * Increased DC initiator default count to 32 for performance optimization
+ * Added ConnectX-8 device support to IB
+ * Improved logs by suggesting to increase PF_LOG_BAR_SIZE when UAR allocation fails
+ * Added support for subnet filter list on RoCE GID
+ * Added a reason when a resource's lane is unreachable
+ * Added IB MLX5 as a separate UCX modules with RPM
+ * Added initial GGA support
+ * Added setting IB DEVX atomic_mode based on device capabilities
+ * Removed DC keepalive
+ * Implemented user-mode registration approach for XGVMI
+#### CUDA
+ * Improved GDR latency estimations on AMD Milan systems
+ * Improved reliability by detecting runtime/build GDR library version match
+ * Added handling missing IPC capability when unpacking keys
+ * Added caching for CUDA IPC memory pool import operation
+ * Added GDR environment variables to optimize performance on Grace Hopper systems
+ * Added CUDA Fabric memory support
+ * Added multi-node NVLINK support
+ * Improved peer accessible cache for CUDA IPC to be reentrant
+ * Added detection and allocation of Fabric VMM memory
```

Comment:
i'd move it to top of this section

Reply:
fixed, reordered some member of this section